package/MDAnalysis/lib/qcprot.pyx

```diff
@@ -181,7 +181,7 @@ def InnerProduct(np.ndarray[np.float64_t,ndim=1] A,
     A[0] = A[1] = A[2] = A[3] = A[4] = A[5] = A[6] = A[7] = A[8] = 0.0
 
     if (weight is not None):
-        for i in xrange(N):
```

Comment:
How is Cython dealing with `range` and `xrange`? Did Cython made the same change as Python regarding how `range` work?